config/settings.yml

```diff
@@ -1433,10 +1433,21 @@ genisis:
   pass: bogus
 
 # Settings for connecting AFS Veteran Services
+# For locahost we can use the existing certs as long as we don't call out
 dgi:
+  # add med_api here? Will need to reach out to that team
```

Comment:
Could we not put comments like this in code but just keep it in your teams backlog to follow up on?

Reply:
comment removed